common/test/acceptance/pages/studio/settings.py

```diff
@@ -234,3 +238,50 @@ def refresh_page(self, wait_for_confirmation=True):
             ).fulfill()
         self.wait_for_require_js()
         self.wait_for_ajax()
+
+    @staticmethod
+    def get_asset_path(file_name):
```

Comment:
Why not just put it in a common place as a utility method? I would invoke the rule of two... never have identical code in two places. :)